### PR TITLE
adding info about Xcode 7.0.1, Xcode 7.1 releases.

### PR DESCRIPTION
**Pull request by emdantrim**
I'm pretty sure that these are all the right simulators and stuff, but perhaps someone like @henrikhodne, @cotsog, or @MariadeAnton would like to look over it beforehand?

### CODE REVIEW

**Comment by solarce** [comment]
Looks great to me

File: user/languages/objective-c.md [hunk 1]
```diff
@@ -23,15 +23,92 @@ NDA imposed on them. We do test them internally, and our goal is to make new
...
-### Xcode 7
+### Xcode 7.1
 
-Xcode 7 GM is available by adding `osx_image: xcode7` to your .travis.yml.
+Xcode 7.1 GM is available by adding `osx_image: xcode7.1` to your .travis.yml.
 
 Our Xcode 7 image has the following SDKs preinstalled:
 
 - macosx10.11
 - iphoneos9.0
```
Comment (cotsog) [inline user/languages/objective-c.md]: `iphoneos9.0` and `iphonesimulator9.0` are not installed, so I would remove them from this list.

`iphoneos9.1` and `appletvsimulator9.0` should be added to the list.

See https://magnum.travis-ci.com/travis-pro/xcode7-sample-project/jobs/25873523#L118

**Comment by solarce** [comment]
@cotsog how's it look now?

File: user/languages/objective-c.md [hunk 2]
```diff
@@ -23,13 +23,90 @@ NDA imposed on them. We do test them internally, and our goal is to make new
...
 
 Our Xcode 7 image has the following SDKs preinstalled:
 
 - macosx10.11
+- iphoneos9.1
+- iphonesimulator9.1
+- tvos9.0
+- watchos2.0
+- watchsimulator2.0
+- appletvsimulator9.0
```
Comment (cotsog) [inline user/languages/objective-c.md]: Nit: I would place this line below `tvos9.0`

File: user/languages/objective-c.md [hunk 3]
```diff
@@ -23,13 +23,90 @@ NDA imposed on them. We do test them internally, and our goal is to make new
...
+  - iPad 2
+  - iPad Retina
+  - iPad Air
+- iOS 9.0
+  - iPhone 4s
+  - iPhone 5
+  - iPhone 5s
+  - iPhone 6
+  - iPhone 6 Plus
+  - iPad 2
```
Comment (cotsog) [inline user/languages/objective-c.md]: Also need to add:

    - iPhone 6s
    - iPhone 6s Plus

File: user/languages/objective-c.md [hunk 4]
```diff
@@ -23,13 +23,90 @@ NDA imposed on them. We do test them internally, and our goal is to make new
...
+  - iPad Retina
+  - iPad Air
+  - iPad Air 2
+- iOS 9.1
+  - iPhone 4s
+  - iPhone 5
+  - iPhone 5s
+  - iPhone 6
+  - iPhone 6 Plus
+  - iPad 2
```
Comment (cotsog) [inline user/languages/objective-c.md]: Also need to add:

    - iPhone 6s
    - iPhone 6s Plus

File: user/languages/objective-c.md [hunk 5]
```diff
@@ -23,13 +23,90 @@ NDA imposed on them. We do test them internally, and our goal is to make new
...
+  - iPad Retina
+  - iPad Air
+  - iPad Air 2
+  - iPad Pro
+- watchOS 2.0
+  - Apple Watch - 38mm
+  - Apple Watch - 42mm
+
```
Comment (cotsog) [inline user/languages/objective-c.md]: Also:

    - tvOS 9.0
      - Apple TV 1080p

**Pull request merged**